.github/workflows/deploy.yml

```diff
@@ -74,6 +74,12 @@ jobs:
       with:
         creds: ${{ secrets.AzureSPN }}
 
+    - name: Get ACR Login Server
+      run: |
+        # Retrieve ACR Login Server value from deployment output
+        ACR_LOGIN_SERVER=$(az deployment group show -g ${{ env.RESOURCE_GROUP_NAME }} -n main --query properties.outputs.acrLoginServer.value | tr -d '"')
```

Comment:
So this is how one sets a variable in a GHA? does one need to declare it first or just throw it out there and it gets saturated?